### PR TITLE
[REVIEW] minor fixes to notebooks with API changes for 0.12

### PR DESCRIPTION
**Pull request by taureandyernv**
To Fix

- [x] conference_notebooks.02-Louvain
- [x] conference_notebooks.03-UMAP
- [x] getting_started_notebooks.02_Introduction_to_cuDF
- [x] getting_started_notebooks.06_Introduction_to_Supervised_Learning
- [x] intermediate_notebooks.linear_regression_demo
- [x] intermediate_notebooks.ridge_regression_demo
- [x] intermediate_notebooks.umap_demo_full
- [x] blog_notebooks.regression_2_blog - awaiting xgboost 0.12 update to test, but should be okay for for 1/24 nightlies

To Archive

- [x] blog_notebooks/cyber
- [x] advanced_notebooks.rapids_customized_kernels - archiving until fixed

To Sunset

- [x] intermediate_notebooks.cuDatashader
- [x] intermediate_notebooks.cuml_benchmarks_quick

To Remove before moving elsewhere

- [x] Colab Notebooks folder

### CODE REVIEW

**Comment by taureandyernv** [comment]
alright, i think its all yours to review, @Salonijain27

**Comment by taureandyernv** [comment]
Slipped in a few goodies, like testing.md, and updated multimedia links

File: intermediate_notebooks/examples/linear_regression_demo.ipynb [hunk 1]
```diff
@@ -1,768 +1,820 @@
...
+   "source": [
+    "# Intro to  Linear Regression with cuML\n",
+    "Corresponding notebook to [*Beginner’s Guide to Linear Regression in Python with cuML*](http://bit.ly/cuml_lin_reg_friend) story on Medium\n",
+    "\n",
+    "Linear Regression is a simple machine learning model where the response `y` is modelled by a linear combination of the predictors in `X`. The `LinearRegression` function implemented in the `cuML` library allows users to change the `fit_intercept`, `normalize`, and `algorithm` parameters. \n",
+    "\n",
+    "Here is a brief on RAPIDS' Linear Regression parameters:\n",
+    "\n",
+    "- `algorithm`: 'eig' or 'svd' (default = 'eig')\n",
+    "    - `Eig` uses a eigendecomposition of the covariance matrix, and is much faster\n",
```
Comment (Salonijain27) [inline intermediate_notebooks/examples/linear_regression_demo.ipynb]: space between eigen decomposition missing

File: intermediate_notebooks/examples/linear_regression_demo.ipynb [hunk 2]
```diff
@@ -1,768 +1,820 @@
...
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "8MqX73B4s5tv",
-        "colab_type": "text"
-      },
-      "source": [
-        "## Multiple Linear Regression \n",
-        "- Our mean squared error for Simple Linear Regression looks kinda high..\n",
```
Comment (Salonijain27) [inline intermediate_notebooks/examples/linear_regression_demo.ipynb]: there is an extra `.` at the end of the sentence

File: intermediate_notebooks/examples/linear_regression_demo.ipynb (hunk 1 above)
Comment (taureandyernv) [reply intermediate_notebooks/examples/linear_regression_demo.ipynb]: fixed

File: intermediate_notebooks/examples/linear_regression_demo.ipynb (hunk 2 above)
Comment (taureandyernv) [reply intermediate_notebooks/examples/linear_regression_demo.ipynb]: fixed

**Comment by taureandyernv** [comment]
waiting on xgboost updates to finish review and merge

**Pull request merged**